plenum/server/database_manager.py

```diff
@@ -82,6 +83,19 @@ def is_taa_acceptance_required(self, lid):
             return False
         return self.databases[lid].taa_acceptance_required
 
+    def set_state_version_controller(self, controller):
+        self._state_version_controller = controller
+
+    def update_state_version(self, txn):
+        if self._state_version_controller is not None:
+            self._state_version_controller.update_version(txn)
```

Comment:
It looks like we assume some interface here for state version controller, but there is no definition of the interface. It would be great to define it in Plenum.